demos/mqtt/mqtt_demo_basic_tls/mqtt_demo_basic_tls.c

```diff
@@ -655,9 +769,18 @@ static void eventCallback( MQTTContext_t * pMqttContext,
         switch( pPacketInfo->type )
         {
             case MQTT_PACKET_TYPE_SUBACK:
-                LogInfo( ( "Subscribed to the topic %.*s.\n\n",
-                           MQTT_EXAMPLE_TOPIC_LENGTH,
-                           MQTT_EXAMPLE_TOPIC ) );
+
+                /* Decode SUBACK and update globalSubAckStatus accordingly.  */
+                updateSubAckStatus( pPacketInfo );
+
+                if( globalSubAckStatus != MQTTSubAckFailure )
```

Comment:
Please add a descriptive comment about what the if-expression means. It can say something like "Check whether the subscription request was successful"